### ISSUE TITLE
Error during generation of java files from multiple schema files

### ISSUE THREAD

**Issue by lbar** [edeandrea/xjc-generation-gradle-plugin#9]
If several schema files are used to describe a set of related entities, the generation fails with this message:
```
[ant:xjc] [ERROR] "file:/dev/project/src/main/xsd/type.xsd" is not part of this compilation. Is it a mistake for "file:/dev/project/src/main/xsd/product.xsd" ?
```
The problem occurs because the plugin process each file separately. All the schema files should be processed in one pass to resolve the issue.

**Comment by lbar** [comment]
I linked my proposal for fix. What do you think about this?

**Comment by edeandrea** [comment]
I added in some comments in the PR

**Comment by edeandrea** [comment]
Thanks for all the help with this! I've released version 1.3 to the plugin portal.
